src/app/store.ts

```diff
@@ -1,6 +1,13 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
```

Comment:
It seems like there might be a typo or incorrect import with `combineSlices`. Typically, Redux Toolkit uses `combineReducers`. Verify if `combineSlices` is the intended function or if it should be `combineReducers`.